README.md

```diff
@@ -14,9 +15,9 @@ Bevy is still in the _very_ early stages of development. APIs can and will chang
 
 ## Design Goals
 
-* **Capable**: Offer a complete 2D and 3D feature set 
+* **Capable**: Offer a complete 2D and 3D feature set
```

Comment:
Bah, my vim removes spaces at the end of lines. Great for code, not as great for READMEs.